tests/src/collect.rs

```diff
@@ -314,12 +319,30 @@ impl<'a> Parser<'a> {
         let kind: NoteKind = head.parse().ok()?;
         self.s.eat_if(' ');
 
+        let mut file = None;
+        if self.s.eat_if('"') {
+            let path = self.s.eat_until('"');
+            let vpath = VirtualPath::new(path);
+            file = Some(FileId::new(None, vpath));
+
+            self.s.eat_if('"');
+            self.s.eat_if(' ');
+        }
```

Comment:
If you ommit the closing `"`, it will complain that there is no range and give you the proper place where you made the error. Not much more I can do I think without complexifying the code a lot, for an error that only happens when writing tests 🤔

Reply:
Actually my bad, I found a really easy way of adding a nicer error 😓